operations/security_and_authentication/Change_the_Keycloak_Admin_Password.md

```diff
@@ -1,46 +1,50 @@
-## Change the Keycloak Admin Password
+# Change the Keycloak Admin Password
 
-Update the default password for the admin Keycloak account using the Keycloak user interface \(UI\). After updating the password in Keycloak, encrypt it on the system and verify that the change was made successfully.
+Update the default password for the admin Keycloak account using the Keycloak user interface (UI). After updating the password in Keycloak, encrypt it on the system and verify that the change was made successfully.
 
-This procedure uses SYSTEM\_DOMAIN\_NAME as an example for the DNS name of the non-compute node \(NCN\). Replace this name with the actual NCN's DNS name while executing this procedure.
+This procedure uses SYSTEM\_DOMAIN\_NAME as an example for the DNS name of the non-compute node (NCN). Replace this name with the actual NCN's DNS name while executing this procedure.
 
-### Procedure
+## Procedure
 
-1.  Log in to Keycloak with the default admin credentials.
+1. Log in to Keycloak with the default admin credentials.
 
-    Point a browser at https://auth.SYSTEM_DOMAIN_NAME/keycloak/admin, replacing SYSTEM\_DOMAIN\_NAME with the actual NCN's DNS name.
+    Point a browser at `https://auth.SYSTEM_DOMAIN_NAME/keycloak/admin`, replacing SYSTEM\_DOMAIN\_NAME with the actual NCN's DNS name.
```

Comment:
Should this line also have cmn. added to the url?

Reply:
cmn is for 1.2. The examples for 0.9 and 1.0 shouldn't have it.